lib/widget/form_textfield.dart

```diff
@@ -100,7 +100,8 @@ abstract class BaseTextInput extends StatefulWidget with Invokable, HasControlle
       'borderRadius': (value) => _controller.borderRadius = Utils.optionalInt(value),
       'validator': (value) => _controller.validator = Utils.getValidator(value),
       'obscureToggle': (value) => _controller.obscureToggle = Utils.optionalBool(value),
-      'keyboardAction': (value) => _controller.keyboardAction = _getKeyboardAction(value)
+      'keyboardAction': (value) => _controller.keyboardAction = _getKeyboardAction(value),
+      'maxLines' : (value) => _controller.updateMaxline(value),
```

Comment:
1. user may enter a string and this will throw an error. Use Utils.optionalInt(value)
2. You shouldn't need to do any of this nor call notifyListeners. Simply setting the maxLine will automatically call notifyListeners().

Reply:
fixed